packages/wdio-browserstack-service/src/launcher.js

```diff
@@ -1,19 +1,27 @@
+import { promisify } from 'util'
+import { performance, PerformanceObserver } from 'perf_hooks'
+
 import BrowserstackLocalLauncher from 'browserstack-local'
 import logger from '@wdio/logger'
 
 const log = logger('@wdio/browserstack-service')
 
 export default class BrowserstackLauncherService {
-    onPrepare(config, capabilities) {
-        if (!config.browserstackLocal) {
+    constructor (options, capabilities, config) {
```

Comment:
Seems like `capabilities` is not used, can we remove it from the constructor?

Reply:
We could rename it to `_` given that it is not important but we use config and therefor need to access the 3rd parameter somehow.